rlog/log.go

```diff
@@ -115,6 +117,17 @@ func (l *defaultLogger) Level(level string) {
 	}
 }
 
+func (l *defaultLogger) OutputPath(path string) (err error) {
+	var file *os.File
+	file, err = os.OpenFile(path, os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0666)
+	if err != nil {
+		return
```

Comment:
Maybe should `return err`

Reply:
Has already return err. err is defined in the api definition.
```go
func (l *defaultLogger) OutputPath(path string) (err error)
```